src/http/vault.ts

```diff
@@ -68,15 +64,6 @@ export class VaultClient extends JsonRpcClient {
         await this.post("request_replace", [request.toHex()]);
     }
 
-    async registerVault(collateral: string, network: bitcoin.Network): Promise<string> {
```

Comment:
We would still want to give the option to register a vault via the UI. So we would need this one in there.

Reply:
Agreed. If `encodeBtcAddress` is the concern, just return an empty promise.

Reply:
I'm returning the JsonRpcResponse just in case we might need it in the future: https://github.com/interlay/polkabtc-js/pull/219/files#diff-923999cc29979ac83f22e93188908d26599e6cdaf298a00258ca5d2dffece334R81

Reply:
Makes sense!